src/stack.js

```diff
@@ -1,5 +1,26 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+  constructor() {
+    this.size = 0
+    this.elements = []
+  }
+
+  push = (elementToAdd) => {
```

Comment:
Consider implementing the `Stack` class using `Node` objects, so that you are required to directly implement the `push` and `pop` functions, instead of delegating to the array class.